gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -304,77 +303,119 @@ You should see a list of services. Flex Gateway is successfully running if each
+...Successfully got an update from the "flex-gateway" chart repository
+Update Complete. ⎈Happy Helming!⎈
 ----
++
+If you have more than one Helm repository on your machine, the message in your terminal window lists all of the repositories.
+// end::k8s-flex-helm-chart-add[]
+// tag::k8s-flex-helm-chart-deploy[]
+//      PLEASE retain blank line before first step below
 
-. Using Ingress, install the *flex-gateway* Helm chart into the *gateway* namespace.
+. Run the Helm command for deploying your gateway and connecting it to Anypoint Platform:
 +
-[source,kubernetes]
+[source,helm]
 ----
-helm -n gateway upgrade -i --create-namespace --wait ingress flex-gateway/flex-gateway \
+helm -n gateway upgrade -i --create-namespace \
+--wait ingress flex-gateway/flex-gateway \ 
+--set gateway.mode=connected \
 --set-file registration.content=registration.yaml
 ----
 +
-The command returns something similar to the following: 
+This command creates the `gateway` namespace and a release named `ingress` if they do not exist. You can use names of your choice for your namespace and release. The command syntax for the Helm repository and chart names is `<helm-repo-name>/<helm-chart-name>`. 
++
+The command uses `--set gateway.mode=connected` because the default for the Helm chart is Local Mode.
++
+When successful, the command prints a message indicating an upgrade to your `ingress` release:
 +
-[source,text]
 ----
+Release "ingress" does not exist. Installing it now.
 NAME: ingress
-LAST DEPLOYED: Tue Oct 19 13:08:07 2021
+LAST DEPLOYED: Mon Mar 20 21:36:19 2023
 NAMESPACE: gateway
 STATUS: deployed
 REVISION: 1
 TEST SUITE: None
 ----
-
-// end::k8s-install-helm-chart[]
++
+The `REVISION` value increments the `ingress` release by `1` each time you run this command with the same namespace, repository, and chart names. For example, if you run the command a second time, you find a new revision number (`REVISION: 2`). The `LAST DEPLOYED` date reflects the date of that revision.
+//    do not add an empty new line here at end, please
+// end::k8s-flex-helm-chart-deploy[]
 // tag::gateway-connected[]
 
-In Runtime Manager, click *&#8592; Flex Gateway* in the left navigation. Notice that your gateway's status is *Connected*. Refresh the page, if necessary.
-
-If a Flex Replica
-in Connected Mode is stopped it will be removed from the UI in Runtime Manager after 30 days. Otherwise,
-it will appear in the Runtime Manager UI even if it is no longer running.
+In Runtime Manager, you can click *&#8592; Flex Gateway* in the left navigation to find your new gateway in the UI. Notice that your gateway's current status is *Connected*. Refresh the page, if necessary.
 
+NOTE::
+After 30 days, a stopped or deleted gateway is removed from the Runtime Manager UI. Otherwise, the UI continues to list the gateway even if it is no longer running or connected.
 // end::gateway-connected[]
 // tag::helm-chart-options[]
 
 [[helm-chart-options]]
-=== Helm Chart Configuration Options
```

Comment:
Technically, the default settings that people look at with the command in this section are configurable. However, you can get the impression that these are setting in YOUR Helm chart for flex, which they won't be if you change the defaults. You can see settings that have been revised chart via a different command. In this section, there will be a link to that info from this section.